app/vlogscli/main.go

```diff
@@ -402,6 +402,11 @@ func getQueryResponse(ctx context.Context, output io.Writer, qStr string, output
 	req.Header.Set("AccountID", strconv.Itoa(*accountID))
 	req.Header.Set("ProjectID", strconv.Itoa(*projectID))
 
+	if err := authConfig.SetHeaders(req, true); err != nil {
```

Comment:
@func25 FYI.
`SetHeaders()` may set `req.Host` to `ac.tlsServerName`.

```go
func (ac *Config) SetHeaders(req *http.Request, setAuthHeader bool) error {
	if ac.tlsServerName != "" {
		// It tlsServerName is set, then it is likely the request is performed via IP address instead of hostname.
		// In this case users expect that the specified tlsServerName is used as a Host header in the request to https server.
		// See https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5802
		req.Host = ac.tlsServerName
	}
```